crates/modelardb_compression/src/compression.rs

```diff
@@ -261,307 +261,354 @@ mod tests {
 
     use super::*;
 
-    use arrow::array::{BinaryArray, Float32Array, UInt64Array, UInt64Builder, UInt8Array};
+    use arrow::array::{
+        ArrayBuilder, BinaryArray, Float32Array, UInt64Array, UInt64Builder, UInt8Array,
+    };
     use modelardb_common::array;
     use modelardb_common::test::data_generation::{self, ValuesStructure};
     use modelardb_common::types::{TimestampBuilder, ValueBuilder};
 
-    use crate::models;
+    use crate::tests::{ERROR_BOUND_FIVE, ERROR_BOUND_ZERO};
+    use crate::{models, MODEL_TYPE_NAMES};
 
-    const ERROR_BOUND_ZERO: f32 = 0.0;
-    const ERROR_BOUND_FIVE: f32 = 5.0;
-    const MULTIPLY_NOISE_RANGE: Option<Range<f32>> = Some(1.0..1.05);
+    const UNIVARIATE_ID: u64 = 1;
+    const ADD_NOISE_RANGE: Option<Range<f32>> = Some(1.0..1.05);
     const TRY_COMPRESS_TEST_LENGTH: usize = 50;
 
     // Tests for try_compress().
     #[test]
-    fn test_try_compress_empty_time_series() {
-        let compressed_record_batch = compress_time_series(
+    fn test_try_compress_absolute_empty_time_series() {
```

Comment:
Maybe rename the tests in this file to make it clear that it is the error bound that is either absolute or relative and not the time series itself.

Reply:
Fixed.